docs/layer.md

```diff
@@ -25,53 +23,56 @@ which helps avoid id collisions in this case.
 
 E.g. assuming a composite GeoJsonLayer layer that renders two sublayers,
 choropleths and lines, with those ids:
-```
-   new GeoJsonLayer({id: 'street-grid'});
-```
+
+    new GeoJsonLayer({id: 'street-grid'});
+
 Will generate the following layers and ids:
-```
-   GeoJsonLayer:'street-grid'
-   ChoroplethLayer:'street-grid-choropleth'
-   LineLayer:'street-grid-choropleth'
-```
+
+    GeoJsonLayer: 'street-grid'
+    ChoroplethLayer: 'street-grid-choropleth'
+    LineLayer: 'street-grid-choropleth'
 
 React Note: `id` is similar to the `key` property used in React to match
 components between rendering calls.
 
+##### `viewport` (Viewport, optional)
 
-### `viewport` (Viewport, optional, default=injected from context)
+- Default: Injected from context
 
 Viewport can be supplied to override the context supplied viewport.
 
+##### `pickable` (Boolean, optional)
 
-### `pickable` [bool, optional, default=false]
+- Default: `false`
 
 Whether layer responds to mouse events.
 
+##### `visible` (Boolean, optional)
+
+- Default: `true`
 
-### `visible` [bool, optional, default=true] whether layer is drawn
+Whether layer is drawn.
 
 For performance reasons it is often better to control layer visibility
 with the `visible` prop rather than through conditional rendering.
 
 Compare:
-```
-   const layers = [
-   	 new MyLayer({visible: showMyLayer});
-   ];
-```
+
+    const layers = [
+      new MyLayer({visible: showMyLayer});
+    ];
+
 with
-```
-   const layers = [];
-   if (showMyLayer) {
-   	 new MyLayer({visible: showMyLayer});
-   }
-```
+
+    const layers = [];
+    if (showMyLayer) {
+      new MyLayer({visible: showMyLayer});
+    }
+
 In the second example (conditional rendering) the layer state will
 be destroyed and regenerated every time the showMyLayer flag changes.
 
-
-### `opacity` (number, required)
+##### `opacity` (Number, required)
```

Comment:
I think the number of hashes `###` is important as it determines the sizing of the header. We should be consistent across files. Suggest using 3 hashes unless visuals are better with 5.